### ISSUE TITLE
9/2 - City state quest for antiquity site

### ISSUE THREAD

**Issue by silentreply** [LoneGazebo/Community-Patch-DLL#7057]
_1. Mod version (i.e Date - 4/17):_ 9-2

_2. Mod list (if using Vox Populi only, leave blank):_

_3. Error description:_ 
This may not be a bug, but I'm just curious if this site would really count for the quest. I was under the impression that they only give this quest if they have a dig site in their borders or immediately adjacent. What counts as an antiquity site "near" a city state for the purpose of fulfilling the quest?
![image](https://user-images.githubusercontent.com/59859911/92830354-37d2c480-f3a3-11ea-8341-d605190b5220.png)

_4. Steps to reproduce (optional):_

---------------------------
Supporting information:
Please note that you can attach .zip files by dragging-and-dropping them. If possible, zip up all supporting data and post that way.

1. Log files (always attach your Logs folder, located at My Documents/My Games/Sid Meier's Civilization 5. Make sure you have enabled logging before experiencing an error! Go here to find out how: http://forums.civfanatics.com/showthread.php?t=487482):

2. Save game (always attach a save that was made a turn before the error; located at My Documents/My Games/Sid Meier's Civilization 5/ModdedSaves):

3. CvMiniDump.dmp file (attach if experiencing a game crash. Located at Program Files/Steam/steamapps/common/Sid Meier's Civilization V):

4. Screenshots (optional):

**Comment by RecursiveVision** [comment]
```
/// Any dig sites near us?
/// NOTE: This should pick a dig deterministically, given the current implementation of distributing global quests
CvPlot* CvMinorCivAI::GetBestNearbyDig()
{
	CvCity* pCapital = GetPlayer()->getCapitalCity();

	// Minor must have Capital
	if(pCapital == NULL)
	{
		return NULL;
	}

	CvWeightedVector<int, 64, true> viPlotIndexes; // 64 camps in 12 hex radius should be enough

	int iRange = 12;

	CvPlot* pLoopPlot;

	// Loop in all plots in range
	int iDX, iDY;
	for(iDX = -(iRange); iDX <= iRange; iDX++)
	{
		for(iDY = -(iRange); iDY <= iRange; iDY++)
		{
			pLoopPlot = plotXY(pCapital->getX(), pCapital->getY(), iDX, iDY);

			if(pLoopPlot != NULL)
			{
				int iDistance = plotDistance(pCapital->getX(), pCapital->getY(), pLoopPlot->getX(), pLoopPlot->getY());

				if(iDistance <= iRange)
				{
					// Dig site here?
					if(pLoopPlot->getResourceType() == GC.getARTIFACT_RESOURCE())
					{
						int iWeight = 1 + (iRange - iDistance); // Closer digs have higher weight
						viPlotIndexes.push_back(pLoopPlot->GetPlotIndex(), iWeight);
					}
				}
			}
		}
	}

	// Didn't find any nearby
	if(viPlotIndexes.size() == 0)
	{
		return NULL;
	}

	// Choose the best plot
	viPlotIndexes.SortItems();
	CvPlot* pBestPlot = GC.getMap().plotByIndex(viPlotIndexes.GetElement(0));

	return pBestPlot;
}
```
tl;dr - They can choose antiquity sites within 12 tiles of their capital, but prefer them to be closer.

**Comment by silentreply** [comment]
Good to know, thanks!
